editor/src/com/rockbite/tools/talos/editor/addons/bvb/SkeletonContainer.java

```diff
@@ -57,11 +57,9 @@ public void setSkeleton(FileHandle jsonHandle, FileHandle atlasHandle) {
         final SkeletonData skeletonData = json.readSkeletonData(jsonHandle);
 
         // before moving forward let's check if things are missing
-        boolean hasGoners = checkForGoners(skeletonData);
+        checkForGoners(skeletonData);
 
-        if(!hasGoners) {
-            configureSkeleton(skeletonData);
-        }
+        configureSkeleton(skeletonData);
```

Comment:
I made it to fetch the first available skin and animation, because after reading data of skeleton container in any way project wants it not to be null. See the other comment.